safe-global/safe-wallet-web
# feat: batch execute flow

## What it solves

Migrating the batch execute flow to the new transaction flow framework.

## How this PR fixes it

The batch execute flow has been ported, as well as removing the old one.

## How to test it

Queue multiple transactions, execute them in a batch and observe a successful transaction.

## Screenshots

![batch execute flow](https://github.com/safe-global/safe-wallet-web/assets/20442784/4ce8dffc-f4d3-4bd4-a5d1-df4291e7da74)

## Checklist
